templates/shared_services/firewall/terraform/firewall.tf

```diff
@@ -26,13 +26,13 @@ resource "azurerm_firewall" "fw" {
   lifecycle { ignore_changes = [tags] }
 }
 
-resource "azurerm_management_lock" "fw" {
-  count      = var.stateful_resources_locked ? 1 : 0
-  name       = azurerm_firewall.fw.name
-  scope      = azurerm_firewall.fw.id
-  lock_level = "CanNotDelete"
-  notes      = "Locked to prevent accidental deletion"
-}
+# resource "azurerm_management_lock" "fw" {
```

Comment:
I guess temp?

Reply:
No, I removed the firewall lock as it isn't that useful anymore (all rules are anyway in the DB).
I just forgot to delete it here. I will do so...